src/ILCompiler.WebAssembly/src/CodeGen/ILToWebAssemblyImporter.cs

```diff
@@ -1247,6 +1247,15 @@ private void ImportReadOnlyPrefix()
 
         private void ImportThrow()
         {
+            var exceptionObject = _stack.Pop();
+
+            if (TrapFunction.Pointer == IntPtr.Zero)
```

Comment:
Rather than copying this logic, can you please factor it out into the property getter or somewhere else? I imagine we'll need trap in another couple of places too

Reply:
Of course, in fact I did it but them I discarded it because I wanted to focus only on the throw method : S